cli/tests/unit/serve_test.ts

```diff
@@ -3715,3 +3716,33 @@ async function curlRequestWithStdErr(args: string[]) {
   assert(success);
   return [new TextDecoder().decode(stdout), new TextDecoder().decode(stderr)];
 }
+
+Deno.test(
+  {
+    ignore: Deno.build.os === "windows",
+    permissions: { run: true, read: true, write: true },
+  },
+  async function httpServerUnixDomainSocket() {
+    const d = deferred();
+    const ac = new AbortController();
+    const filePath = tmpUnixSocketPath();
+    const server = Deno.serve(
+      {
+        signal: ac.signal,
+        path: filePath,
+        onListen(info) {
```

Comment:
I forgot about this in the original description, but we should also be checking that the second argument to this function is a `Deno.UnixAddr`, ie: https://deno.land/api@v1.37.1?s=Deno.UnixAddr